datalad_container/containers_run.py

```diff
@@ -131,14 +132,41 @@ def __call__(cmd, container_name=None, dataset=None,
             # just prepend and pray
             cmd = container['path'] + ' ' + cmd
 
+        extra_inputs_ = container.get("extra-input",[])
+        # TODO: Can't use EnsureListOf(str) yet as it handles strings as iterables.
+        # See this PR: https://github.com/datalad/datalad/pull/7267
+        # get_all=True still returns a string for a single value
+        extra_inputs_ = [extra_inputs_] if isinstance(extra_inputs_, str) else extra_inputs_
```

Comment:
in principle, we have `ensure_list` which does that  https://github.com/datalad/datalad/blob/maint/datalad/utils.py#L734  so could just wrap around above `container.get` and avoid needing extra `extra_inputs_`